### PR TITLE
my info and picture

### PR DESCRIPTION
**Pull request by emran-ai**
### General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [x] is linked to an issue (if it is related)
- [x] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work
- [x] any embedded code snippets are formatted

### CODE REVIEW

**Review by bermarte — CHANGES_REQUESTED** [review]
Be sure to pass the CI check, then we will merge it.

**Comment by talmurshidi** [comment]
Please create a `images` folder inside `student-bios` folder then you can add your photo inside images folder and delete it from assets folder.
Please make sure to update your branch using `git pull origin main` every time before pushing your changes.

**Review by bermarte — CHANGES_REQUESTED** [review]
Hi, you should not change the README file, be sure you add one file at a time to avoid this error.

File: student-bios/emran-ai.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
 # Introduction
 
-Hi, everyone! My name is Emran and I'm a student. I'm excited to be part of this class and to learn from all of you.
+Hi, everyone! My name is Emran and I'm a student. I'm excited to be part of this
```
Comment (bermarte) [inline student-bios/emran-ai.md:3]: I am a bit confused about this commit, the message says 
> add item to .c spell.json
but you just changed the link to your image (line 16)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <!-- BEGIN TOP -->
```
Comment (bermarte) [inline README.md:1]: You should restore the original README, have a look to [the documentation](https://stackoverflow.com/questions/307579/how-do-i-copy-a-version-of-a-single-file-from-one-git-branch-to-another)

**Review by bermarte — CHANGES_REQUESTED** [review]
https://stackoverflow.com/questions/307579/how-do-i-copy-a-version-of-a-single-file-from-one-git-branch-to-another

**Review by bermarte — CHANGES_REQUESTED** [review]
Please restore the previous README.

**Pull request merged**